services/lsst-condor.rst

```diff
@@ -1,7 +1,15 @@
+:orphan:
+
 ###################################################################
 Using the HTCondor DAC Cluster
 ###################################################################
 
+This content has moved: :doc:`/services/batch`.
+
+The material presented below is for historical reference and will be removed in the future.
+
+------------
+
```

Comment:
@jakerundall Can we go ahead and remove all the 'content' below from this page now?